src/js/health-records/containers/Main.jsx

```diff
@@ -173,12 +178,20 @@ export class Main extends React.Component {
     const customDateRangeError =
       this.state.invalidStartDateFormat ||
       this.state.invalidEndDateFormat;
+    const missingDateError =
+      this.state.missingStartDate ||
+      this.state.missingEndDate;
 
     const customDateOptionClass = classNames({
       'custom-date-option': true,
       'date-range-error': customDateRangeError
     });
 
+    const validationErrorMessages = {
+      rangeError: 'Enter dates in the MM/DD/YYYY date format',
```

Comment:
`customDateRangeError` and `validationErrorMessages.rangeError` seem less descriptive now that there are two types of errors that could occur with date range. Maybe something like `invalidFormatError` would be better now.